test/integration/consul-container/go.mod

```diff
@@ -2,11 +2,14 @@ module github.com/hashicorp/consul/test/integration/consul-container
 
 go 1.19
 
+replace github.com/hashicorp/consul/api/v118 => github.com/hashicorp/consul/api v1.18.0
+
 require (
 	github.com/docker/docker v20.10.11+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/hashicorp/consul v1.14.1
 	github.com/hashicorp/consul/api v1.18.0
+	github.com/hashicorp/consul/api/v118 v118.0.0-00010101000000-000000000000
```

Comment:
I don't know why the line above does not result in `api/v1.18.0` from being used, but it doesn't, the `replace` directive appears to be required.